docs/fabric-native-components.md

```diff
@@ -76,6 +76,23 @@ export interface NativeProps extends ViewProps {
 export default codegenNativeComponent<NativeProps>(
   'CustomWebView',
 ) as HostComponent<NativeProps>;
+import type {HostComponent, ViewProps} from 'react-native';
+import type {BubblingEventHandler} from 'react-native/Libraries/Types/CodegenTypes';
+import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
+
+type WebViewScriptLoadedEvent = {
+  result: 'success' | 'error';
+};
+
+export interface NativeProps extends ViewProps {
+  sourceURL?: string;
+  onScriptLoaded?: BubblingEventHandler<WebViewScriptLoadedEvent> | null;
+}
+
+export default codegenNativeComponent<NativeProps>(
+  'CustomWebView',
+) as HostComponent<NativeProps>;
+
```

Comment:
Added a newline at the end of file to satisfy the linter that comes with React Native community CLI